### PR TITLE
Test query.time('now')

### PR DESCRIPTION
**Pull request by andy-faunadb**

### CODE REVIEW

File: tests/query_test.py [hunk 1]
```diff
@@ -305,6 +305,14 @@ def test_time(self):
     time = "1970-01-01T00:00:00.123456789Z"
     assert self._q(query.time(time)) == FaunaTime(time)
 
+    # 'now' refers to the current time.
+    now = self._q(query.time('now'))
+    assert 0 < (datetime.now() - now.to_datetime()).microseconds < 1000000
```
Comment (ashfire908) [inline tests/query_test.py]: What is this specifically trying to assert?

Comment (andy-faunadb) [inline tests/query_test.py]: That it's reasonably 'now', as opposed to 100 years ago or something.

Comment (freels) [inline tests/query_test.py]: There's no need to test core's behavior here, as long as you get a timestamp back.

Comment (ashfire908) [inline tests/query_test.py]: Matt just said what I was writing a comment about, but I will point out stuff that's wrong with the code assuming time is perfect between machine (which it's not), for future reference

`microseconds` is only the microseconds portion of the delta. You would have to use `total_seconds()` instead to get the actual full difference. Also, `total_seconds()` will be negative if the local clock is running ahead. Also, I believe you would have to use `.utcnow()` as the dates returned from fauna are UTC (unless the comparison is aware of the timezone difference?)

File: tests/query_test.py [hunk 2]
```diff
@@ -305,6 +305,9 @@ def test_time(self):
     time = "1970-01-01T00:00:00.123456789Z"
     assert self._q(query.time(time)) == FaunaTime(time)
 
+    # 'now' refers to the current time.
+    assert isinstance(self._q(query.time('now')), FaunaTime)
```
Comment (ashfire908) [inline tests/query_test.py]: Quote consistency

**Comment by ashfire908** [comment]
LGTM

**Pull request merged**